.golangci.yaml

```diff
@@ -7,7 +7,7 @@ linters-settings:
   goheader:
     values:
       const:
-       - LICENSE: BSD-3-Clause
+        - LICENSE: BSD-3-Clause
```

Comment:
Did you mean to include these changes? They should not be needed.

Reply:
Nah, if some yaml auto-formatter does this, chances are it will just happen again in the future. Doesn't hurt to change it, so let's keep it.